src/main/java/com/uber/jenkins/phabricator/CommentBuilder.java

```diff
@@ -89,9 +95,16 @@ public void processParentCoverage(CodeCoverageMetrics parentCoverage, String bas
             comment.append("Coverage remained the same (" + lineCoverageDisplay + "%)");
         }
 
+        // If coverage change is less than zero and dips below a certain threshold fail the build
+        if (coverageDelta < 0 && coverageDelta < maximumCoverageDecreaseInPercent) {
```

Comment:
Consider this scenario:

coverageDelta = -0.001
maximumCoverageDecreaseInPercent = 0.01

This fails. In fact, this check always fails as long as coverageDelta is negative. I think you want the absolute value of coverageDelta and use that for comparison. Or.. just compare (coverageDelta * -1.0) vs maximumCoverageDecreaseInPercent